Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM elixir:1.5
```

Comment:
FYI 1.6 is out

Reply:
I stuck to the version used in plan.sh, assuming that's what we actually use https://github.com/mozilla/reticulum/blob/master/habitat/plan.sh#L21

Reply:
ah yes, waiting on habitat to upgrade